docs/tutorial.rst

```diff
@@ -1224,65 +1224,54 @@ of at least 0.8 by the end. Next, we'll walk through the steps required to do th
     this conditioning works by re-running the simulation until the condition is
     achieved, a nearly impossible condition will result in very long run times.
 
-First, we need to define the mutation type for the selected mutation.
-So we can refer to it later, we need its "mutation type ID". This is just
-the index of the new mutation type in the contig's list of mutation types.
+First, we need to add a single site mutation in the middle of the contig.
+This represents a mutation of a single base pair, with associated fitness and
+dominance coefficients.  The single site mutation must have a unique string ID
+so that we can refer to it later.
 
 .. code-block:: python
 
-    mt = stdpopsim.MutationType(
-        distribution_type="f",
-        dominance_coeff=1.0,
-        distribution_args=[0.5],
-        convert_to_substitution=False,
-    )
-    dfe = stdpopsim.DFE(
-        id="new_mutation",
-        mutation_types=[mt],
-        proportions=[1.0],
-        description="added mutation",
-        long_description="mutation type to be added",
-    )
-    contig.add_dfe(
-        intervals=np.empty((0, 2), dtype="int"),
-        DFE=dfe,
+    mut_id = "hard sweep"
+    coordinate = round(contig.recombination_map.sequence_length / 2)
+    contig.add_single_site(
+        id=mut_id,
+        coordinate=coordinate,
+        fitness_coeff=0.1,
+        dominance_coeff=0.5,
     )
-    for mt_info in contig.mutation_types():
-        if mt_info["dfe_id"] == dfe.id:
-            break
 
-    mut_id = mt_info["id"]
+.. note::
+
+    Note that single site mutations are internally stored as DFEs, and DFEs
+    cannot overlap. As a consequence, if a DFE is added to the contig over an
+    interval that contains an existing single site mutation, the latter will be
+    "overwritten" and an error will be raised by the simulation engine.
```

Comment:
Are you sure it will raise an error? I don't think that it should - it should just remove the effects of any previous DFEs at that site (and, so maybe it's worth noting you can't put more than one of these single site things at the same position).

Reply:
Oh, I see what error you're talking about, sorry. I read this the wrong way around. Maybe just replace "the latter" with "the single site mutation", to make it harder to get mixed up. (I had read it as saying you couldn't put a single site over the top of an existing DFE.)